### PR TITLE
SFI - ROPC: First group of updates

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #41954

(Hide whitespace changes.)

- Added an include file with the recommendation to use managed identities where possible.
- Other cleanup according to Learn Linter.
- Replace .NET Framework 4.6.1 name include file.
- Remove unnecessary explicit bookmarks.
- Format stuff as code.
- Format tables.

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/additional-tools/self-signed-certificates-guide.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/core/additional-tools/self-signed-certificates-guide.md) | [docs/core/additional-tools/self-signed-certificates-guide](https://review.learn.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide?branch=pr-en-us-42254) |
| [docs/core/porting/modernize.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/core/porting/modernize.md) | [docs/core/porting/modernize](https://review.learn.microsoft.com/en-us/dotnet/core/porting/modernize?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/wcf/add-of-commonparameters.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/wcf/add-of-commonparameters.md) | [docs/framework/configure-apps/file-schema/wcf/add-of-commonparameters](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/add-of-commonparameters?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/wcf/commonparameters.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/wcf/commonparameters.md) | [docs/framework/configure-apps/file-schema/wcf/commonparameters](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/commonparameters?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/wcf/serviceauthorization-element.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/wcf/serviceauthorization-element.md) | [docs/framework/configure-apps/file-schema/wcf/serviceauthorization-element](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/serviceauthorization-element?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/wcf/workflowruntime.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/wcf/workflowruntime.md) | [docs/framework/configure-apps/file-schema/wcf/workflowruntime](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/workflowruntime?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow.md) | [docs/framework/configure-apps/file-schema/windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/windows-workflow-foundation/sqlworkflowinstancestore.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/windows-workflow-foundation/sqlworkflowinstancestore.md) | [docs/framework/configure-apps/file-schema/windows-workflow-foundation/sqlworkflowinstancestore](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/windows-workflow-foundation/sqlworkflowinstancestore?branch=pr-en-us-42254) |
| [docs/framework/configure-apps/file-schema/windows-workflow-foundation/system-servicemodel-of-workflow.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/configure-apps/file-schema/windows-workflow-foundation/system-servicemodel-of-workflow.md) | [docs/framework/configure-apps/file-schema/windows-workflow-foundation/system-servicemodel-of-workflow](https://review.learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/windows-workflow-foundation/system-servicemodel-of-workflow?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/configuring-tracking-for-a-workflow.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/configuring-tracking-for-a-workflow.md) | [docs/framework/windows-workflow-foundation/configuring-tracking-for-a-workflow](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/configuring-tracking-for-a-workflow?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/csharp-expressions.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/csharp-expressions.md) | [docs/framework/windows-workflow-foundation/csharp-expressions](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/csharp-expressions?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/dynamic-update.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/dynamic-update.md) | [docs/framework/windows-workflow-foundation/dynamic-update](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/dynamic-update?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/exceptions.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/exceptions.md) | [docs/framework/windows-workflow-foundation/exceptions](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/exceptions?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/how-to-deserialize-instance-data-properties.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/how-to-deserialize-instance-data-properties.md) | [docs/framework/windows-workflow-foundation/how-to-deserialize-instance-data-properties](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/how-to-deserialize-instance-data-properties?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md) | [docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md) | [docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/how-to-update-the-definition-of-a-running-workflow-instance.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/how-to-update-the-definition-of-a-running-workflow-instance.md) | [docs/framework/windows-workflow-foundation/how-to-update-the-definition-of-a-running-workflow-instance](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/how-to-update-the-definition-of-a-running-workflow-instance?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/net-framework-3-0-wf-in-net-framework-4-interop.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/net-framework-3-0-wf-in-net-framework-4-interop.md) | [docs/framework/windows-workflow-foundation/net-framework-3-0-wf-in-net-framework-4-interop](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/net-framework-3-0-wf-in-net-framework-4-interop?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/pausing-and-resuming-a-workflow.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/pausing-and-resuming-a-workflow.md) | [docs/framework/windows-workflow-foundation/pausing-and-resuming-a-workflow](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/pausing-and-resuming-a-workflow?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/pick-activity.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/pick-activity.md) | [docs/framework/windows-workflow-foundation/pick-activity](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/pick-activity?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/primitives-activities-in-wf.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/primitives-activities-in-wf.md) | [docs/framework/windows-workflow-foundation/primitives-activities-in-wf](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/primitives-activities-in-wf?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/required-arguments-and-overload-groups.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/required-arguments-and-overload-groups.md) | [docs/framework/windows-workflow-foundation/required-arguments-and-overload-groups](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/required-arguments-and-overload-groups?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/runtime-activities-in-wf.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/runtime-activities-in-wf.md) | [docs/framework/windows-workflow-foundation/runtime-activities-in-wf](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/runtime-activities-in-wf?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/samples/database-access-activities.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/samples/database-access-activities.md) | [docs/framework/windows-workflow-foundation/samples/database-access-activities](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/samples/database-access-activities?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/sql-workflow-instance-store.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/sql-workflow-instance-store.md) | [docs/framework/windows-workflow-foundation/sql-workflow-instance-store](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/sql-workflow-instance-store?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/store-extensibility.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/store-extensibility.md) | [docs/framework/windows-workflow-foundation/store-extensibility](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/store-extensibility?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/tracking-events-reference.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/tracking-events-reference.md) | [docs/framework/windows-workflow-foundation/tracking-events-reference](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/tracking-events-reference?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/tracking-participants.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/tracking-participants.md) | [docs/framework/windows-workflow-foundation/tracking-participants](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/tracking-participants?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/variable-and-argument-tracking.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/variable-and-argument-tracking.md) | [docs/framework/windows-workflow-foundation/variable-and-argument-tracking](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/variable-and-argument-tracking?branch=pr-en-us-42254) |
| [docs/framework/windows-workflow-foundation/workflow-tracking-and-tracing.md](https://github.com/dotnet/docs/blob/d1b3cc193092d6e03a146a94e6781d7e9fddfd34/docs/framework/windows-workflow-foundation/workflow-tracking-and-tracing.md) | [docs/framework/windows-workflow-foundation/workflow-tracking-and-tracing](https://review.learn.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/workflow-tracking-and-tracing?branch=pr-en-us-42254) |

</details>

> [!NOTE]
> This table shows preview links for the 30 files with the most changes. For preview links for other files in this PR, select <strong>OpenPublishing.Build Details</strong> within [checks](https://github.com/dotnet/docs/pull/42254/checks).

### CODE REVIEW

File: docs/framework/includes/managed-identities.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> Microsoft recommends that you use the most secure authentication flow available. If you're connecting to Azure SQL, [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication) is the recommended authentication method.
```
Comment (gewarren) [inline docs/framework/includes/managed-identities.md:2]: Here's the new include file.

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 2]
```diff
@@ -115,6 +114,8 @@ The steps in this section add properties and helper methods to the form class th
 
 3. Add the following member declarations to the **WorkflowHostForm** class.
 
+   [!INCLUDE [managed-identities](../includes/managed-identities.md)]
```
Comment (gewarren) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md:117]: Example of including the new include file.

File: docs/framework/includes/managed-identities.md (hunk 1 above)
Comment (wadepickett) [reply docs/framework/includes/managed-identities.md:2]: I will create the same include for aspnetcore.docs and aspnetdocs by the same name.  Thanks for leading the way!

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

Let's :shipit:

**Comment by gewarren** [comment]
Thanks @BillWagner!

**Pull request merged**